keps/sig-scheduling/785-scheduler-component-config-api/README.md

```diff
@@ -82,6 +86,16 @@ For the most part, `kubescheduler.config.k8s.io/v1beta1` will be a copy of
 - [ ] `.profiles[*].plugins.unreserve` will be removed.
 - [ ] Embedded types of `RequestedToCapacityRatio` will include missing json tags
   and will be decoded with a case-sensitive decoder.
+  
+The second iteration, `kubescheduler.config.k8s.io/v1beta2`, includes the following changes:
+  - Plugin removals:
+    - `NodeLabel`
+    - `ServiceAffinity`
+    - `NodePreferAvoidPods`
+    - `NodeResourcesLeastAllocated`
+    - `NodeResourcesMostAllocated`
+    - `RequestedToCapacityRatio`
```

Comment:
can we add a short comment for each item? like 

```
...
- `NodeResourcesLeastAllocated` (in favor of `NodeResourcesFit` plugin with a "LeastAllocated" scoring strategy)
...
```

Reply:
Done